src/js/components/Spinner/stories/Simple.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+import { grommet, Box, Grommet, Spinner, Text } from 'grommet';
+
+export const Simple = () => (
+  <Grommet theme={grommet} full>
+    <Box align="center" direction="row" gap="small" pad="small">
+      <Spinner />
+      <Text>Loading...</Text>
+    </Box>
+  </Grommet>
+);
+
+export default {
+  title: 'Utilities/Spinner/Simple',
```

Comment:
I wonder if these should be in Visualizations instead, most of what's in Utilities is non-visual.